dnf-behave-tests/fixtures/gpgkeys/sign.sh

```diff
@@ -24,34 +24,43 @@ for KEY_NAME in $KEYSPECS; do
     KEY_DIR="${DIR}/keys/${KEY_NAME}"
     mkdir "${KEY_DIR}"
 
+    # workaround for gpgme unable to handle long paths
+    # * create a temp directory /tmp/<tempdir>
+    # * symlink key directory -> /tmp/<tempdir>/gpghome
```

Comment:
Can you add a link to the bug in the comment? Or, alternatively, in the commit message.
https://bugzilla.redhat.com/show_bug.cgi?id=1813705